### ISSUE TITLE
[Feature] Choose spawns version

### ISSUE THREAD

**Issue by Arutosio** [modernuo/ModernUO#512]
possibility to choose the version of the spawns.
choose the version of the spawns based on the versions of spawns in the path containing the spawns.

![image](https://user-images.githubusercontent.com/30377429/107997009-1923ca00-6fe2-11eb-945f-fbf0d27357df.png)

**Comment by kamronbatman** [comment]
Yeah that is a good question. So I only have one set of spawns, and I cannot claim they are the best. I think we should get a good set of spawns for each era, broken out by location (and maybe type).
I can build a gump to do this and open it from admin or a command.

How does that sound?

**Comment by Arutosio** [comment]
This is a good idea
